packages/verify-env/src/test/file.ts

```diff
@@ -0,0 +1,7 @@
+const allChains = process.env.REACT_APP_ENV === "development";
```

Comment:
Nit: consider naming this in a way that makes it clear that it's a boolean, see e.g. the `isXYZ` convention in Chakra. `doesXYZ` or `shouldXYZ` are good too.

Reply:
done.